<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>370</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="storeID">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>   매장 아이디 입력</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="storePW">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>   매장 비밀번호 입력</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btn_Accept">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>470</y>
       <width>380</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>수락</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Refuse">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>470</y>
       <width>380</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>거절</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>771</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>주문서</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>91</y>
       <width>771</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
